documentation/src/main/markdown/currentreleasenotes.md

```diff
@@ -7,6 +7,13 @@
 * [solution_name] now supports binary scanning of files up to 100GB via a chunking method employed during upload.    
     <note type="note">This feature requires [blackduck_product_name] 2024.7.0 or later.</note>
 
+### Dependency updates
```

Comment:
Dependency updates should come after "Changed features" and "Resolved issues". ; )

Reply:
Done!